pims/cine.py

```diff
@@ -22,32 +24,41 @@
 from threading import Lock
 import datetime
 import hashlib
+from os.path import split, join, splitext
+import os
+import subprocess as sbp
+from skimage import io, exposure
```

Comment:
If at all possible, `skimage` should remain an *optional* dependency of PIMS.

Reply:
Removed.